src/main/java/com/google/cloud/spanner/pgadapter/wireoutput/WireOutput.java

```diff
@@ -64,7 +65,7 @@ public void send() throws Exception {
    * efficient for responses that contain multiple parts, such as query results.
    */
   public void send(boolean flush) throws Exception {
-    logger.log(Level.FINE, this::toString);
+    logger.log(Level.FINEST, Logging.format("Send", this::toString));
```

Comment:
why has this been reduced ?

Reply:
This is debug information. It says that a PG wire-protocol message has been sent from PGAdapter to the client.